README.md

```diff
@@ -61,3 +61,21 @@ Convert a JSON representation back into an immutable object
 ### `transit.withFilter(function) => transit`
 
 Create a modified version of the transit API that deeply applies the provided filter function to all immutable collections before serialising. Can be used to exclude entries.
+
+### `transit.withRecords([recordClasses])[.withFilter(function)] => transit`
```

Comment:
Yeah I'm sure there's a good way to do that. It's getting late here in Spain and my brain stops working so well. :P Let me think about the best way to implement this.